source/mongodb/internal-database-usage.txt

```diff
@@ -70,6 +78,14 @@ App Services stores a information about unsyncable documents in the
    You can read from the ``__realm_sync.unsynced_documents`` collection,
    but you should not modify it in any way.
 
+If Device Sync is paused due to hitting the 100,000 unsyncable document limit,
+you have two options:
+   
+- :ref:`terminate and re-enable sync <terminating-realm-sync>`, even though
+   the change you're making is a non-breaking change. 
```

Comment:
We should probably omit the info about breaking changes here, as I don't think it's relevant to the info in this section.

```suggestion
- :ref:`terminate and re-enable sync <terminating-realm-sync>`. 
```